docs/operate/get-started/other-hardware/micro-module.md

```diff
@@ -204,158 +34,89 @@ To create a new module compatible with the Micro-RDK, follow these steps.
-1. Select the part status dropdown to the right of your machine's name on the top of the page: {{<imgproc src="configure/machine-part-info.png" resize="500x" declaredimensions=true alt="Restart button on the machine part info dropdown" class="shadow" >}}
-1. Click the copy icon underneath **Machine cloud credentials**.
-   The Micro-RDK needs this JSON object, which contains your machine part secret key and cloud app address, to connect to the [Viam app](https://app.viam.com).
-1. Navigate to the directory of the project you just created.
-1. Create a new <file>viam.json</file> file and paste the machine cloud credentials from the Viam app in.
-1. Save the file.
+    // Required to convert typed readings to generic readings
+    impl Readings for MySensor {
+        fn get_generic_readings(&mut self) -> Result<GenericReadingsResult, SensorError> {
+            Ok(self
+                .get_readings()?
+                .into_iter()
+                .map(|v| (v.0, SensorResult::<f64> { value: v.1 }.into()))
+                .collect())
+        }
+    }
 
-### Error: failed to run custom build command for `esp32-explorer (/host)`
+    // Basic status implementation
+    impl Status for MySensor {
+        fn get_status(&self) -> Result<Option<micro_rdk::google::protobuf::Struct>, StatusError> {
+            Ok(Some(micro_rdk::google::protobuf::Struct {
+                fields: HashMap::new(),
+            }))
+        }
+    }
 
-This may occur for various reasons such as your machine cloud credentials, WiFi SSID, or password not being populated.
-Check that your machine cloud credentials are provided in your project directory as <file>viam.json</file> and ensure that your WiFi credentials are provided.
+   ```
 
-### Error: invalid value `460800` for `--before <BEFORE>`
+   {{< /expand >}}
 
-Change `"-b"` to `"-B` in the <file>Makefile</file>, as `"-B"` is the Baudrate config.
-Run the following commands to flash <file>esp32-server.bin</file> to your ESP32 microcontroller at a high baud rate, wait for 2 seconds, and observe the device's output:
+   For more examples, see the [example module implementation walkthrough](https://github.com/viamrobotics/micro-rdk/blob/main/examples/modular-drivers/README.md).
 
-```sh {class="command-line" data-prompt="$"}
-espflash write-bin 0x0 target/esp32-server.bin -B 460800  && sleep 2 && espflash monitor
-```
+For further details on Micro-RDK development, including credentials management and developer productivity suggestions, please see the [development technical notes page on GitHub](https://github.com/viamrobotics/micro-rdk/blob/main/DEVELOPMENT.md).
 
-You can find additional assistance in the [Troubleshooting section](/manage/troubleshoot/troubleshoot/).
+## Use your module
 
-{{< snippet "social.md" >}}
+To use your module with your ESP32, follow the [Build and flash custom firmware](/operate/get-started/setup-micro/#build-and-flash-custom-firmware) workflow in a different directory from your module.
```

Comment:
```suggestion
To use your module with your ESP32, follow the [Build and flash custom firmware](/operate/get-started/setup-micro/#build-and-flash-custom-firmware) workflow in a separate directory.
```

[opt] 'different directory from your module [directory]' is a bit wordy, I wonder if this conveys the same idea in fewer words? Up to you.